packages/babel-helper-module-transforms/src/index.ts

```diff
@@ -572,11 +575,16 @@ function buildInitStatement(
         NAME: exportName,
         VALUE: acc,
       };
+
+      if (shouldUseDefineProperty.has(exportName)) {
```

Comment:
Nit: Can we just compare with `__proto__`? We can always refactor it later when there are other property names to be handled.